product_docs/docs/biganimal/release/free_trial/quickstart.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: "Quickstart for the BigAnimal free trial"
+navTitle: "Quickstart" 
+tags:
+  - azure
+  - biganimal
+  - postgresql
+  - psql
+---
+
+This topic provides a quick walkthrough of the steps needed to configure a PostgreSQL 14 cluster on BigAnimal running against AWS with a free trial account. 
+
+## Step 1: create an account and sign into the portal
+
+If you haven't done so already, you'll need to [create your EDB account](detail/create_an_account). 
+
+Then, use your account to access the [BigAnimal portal](https://portal.biganimal.com/).
+
+## Step 2: create an AWS cluster 
+
+1. Select the **Clusters** link on the left to navigate to the [Clusters](https://portal.biganimal.com/clusters) page.
+
+2. Select the **Create New Cluster** button.
+
+    You should now find yourself at <https://portal.biganimal.com/create-cluster>.
+
+3. Enter "AWS PostgreSQL cluster" in the **Cluster Name** field.
+4. Enter a strong, memorable password in the **Password** field. 
+
+   This is the admin password for the cluster.
+5. Choose "PostgreSQL" for **Database Type**
+
+   This gives you an official build of PostgreSQL.
+6. Choose "14" for **Postgres Version**.
+4. Select AWS for **Provider**.
+5. Select "US East 1" for **Region**.
+6. For **Instance Type** and **Storage**, only one set of options are available in the trial - select them.
+7. Select **Create Cluster**.
+
+### Provisioning, active clusters and deletion
```

Comment:
I find this heading a little confusing as you aren't explicitly saying anything deletion. How about you move the first sentence to step 6 (You are brought back to the clusters page with your ...) and calling the section Provisioning and starting the clock? or just Provisioning?